Tests/KituraTests/KituraTestBuilder.swift

```diff
@@ -84,6 +86,16 @@ class ServerTestBuilder: RequestTestBuilder, AssertionTestBuilder {
                 })
             }
         }
+        
+        init(_ test: KituraTest, _ method: String, _ path: String, _ URLEncodedString: String) {
```

Comment:
Sorry, can this be urlEncodedString not URLEncodedString, otherwise it looks like a reference to a type.